src/devices/somfy.ts

```diff
@@ -101,4 +101,33 @@ export const definitions: DefinitionWithExtend[] = [
         description: "Tilt and lift blinds motor",
         extend: [m.windowCovering({controls: ["lift", "tilt"]}), m.battery(), m.identify()],
     },
+    {
+        zigbeeModel: ["1871215B"],
+        model: "1871215B",
+        vendor: "Somfy",
+        description: "Connected plug E type with power monitoring",
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering],
```

Comment:
```suggestion
        extend: [m.onOff(), m.electricityMeter()],
```

and all the code below can be removed.